packages/edit-site/src/components/global-styles/font-library-modal/installed-fonts.js

```diff
@@ -437,8 +437,7 @@ function InstalledFonts() {
 						{ isInstalling && <ProgressBar /> }
 						{ shouldDisplayDeleteButton && (
 							<Button
-								// TODO: Switch to `true` (40px size) if possible
-								__next40pxDefaultSize={ false }
+								__next40pxDefaultSize
```

Comment:
This can be seen when you visit, site-editor --> styles --> typogrpahy --> open font library modal and choose font to edit.

| Before | After |
| :-------- | :------- |
| <img width="404" alt="installed-fonts-delete-before" src="https://github.com/user-attachments/assets/78b49672-e399-4526-b464-9f7679e87455"> | <img width="419" alt="Screenshot 2024-09-12 at 11 04 50 AM" src="https://github.com/user-attachments/assets/d8275249-2d4b-483c-b25d-64f78572665b"> |